### ISSUE TITLE
Features to add

### ISSUE THREAD

**Issue by Walkman100** [Walkman100/BasicBrowser#3]
- [x] Command line args
  - [x] single instance
    - [x] open in new tab
  - [x] Reg key to add BasicBrowser to list of `Open With` programs to open `.html`
    - [x] Installer script that installs reg key
- [x] Fix closing tabs (close selected tab & destroy webbrowser)
- [x] Fix tab name change updates
- [x] New and Close window items
  - [ ] Fix opening new window
- [ ] Right click menu on tabs
  - [ ] Close tab
  - [ ] Open new tab
  - [ ] Duplicate tab
  - [ ] Middle-click on a tab to close
- [ ] Fix history dropdown
- [x] Make Edit and help menus do something (#2)
  - [x] Make copy, paste e.t.c. work on URL bar
- [x] Right-click menu on browser (included by default)
- [ ] Middle-click: open link in new tab

**Comment by jackwilsdon** [comment]
Another nice feature would be middle click to open a new tab when hovering over a link, instead of scrolling.

**Comment by Walkman100** [comment]
Unfortunately that functionality lies in the WebBrowser control I'm using. at the moment the only way to open a link in a new tab is Right-Click, Copy, open a new tab, Paste. I'll see what I can do, but I doubt it's possible. Maybe after @deavmi adds the GeckoFX browser (#1) it might be possible.

**Comment by Walkman100** [comment]
Unfortunately "Clicks" aren't registered on .Net WebBrowsers, I tried the code 

``` vb
AddHandler WebBrowser.MouseUp, AddressOf ToolStripGo_Click
```

(For a new tab) and it compiles, but at runtime if I try to open a new tab it gives an error: `System.NotSupportedException: Event MouseUp is not valid on this ActiveX control.`

**Comment by Walkman100** [comment]
Right-click -> open new tab is definitely supported with the GeckoWebBrowser although it may require a lot of work to get right (i.e. time), and I don't know if registering a middle-click is possible, but it probably is :D

**Comment by Walkman100** [comment]
Okay I registered the click event of the WebBrowsers to a sub and added code to check for a middle-click, but that code never runs, it seems as if the click event isn't raised. (I tried all the other mouse events, still nothing. `Click`, `MouseClick`, `MouseDown`, `MouseUp`)
